ops/app/oci-images.nix

```diff
@@ -0,0 +1,24 @@
+let
+  inherit (inputs) std self;
+
+  # TODO: express this as OCI labels (what they are for)
+  BUILD_INFO = builtins.toJSON {
+    inherit (self) lastModified lastModifiedDate rev;
+    shortRev = self.shortRev or "no rev";
+    extra = {
+      sourceInfo = self;
+      narHash = self.narHash;
+      path = self.outPath;
+    };
+  };
+in {
+  server = std.lib.ops.mkStandardOCI {
+    # TODO: set up Repo w/ registry, and test
+    name = "926093910549.dkr.ecr.us-east-1.amazonaws.com/provider-server";
```

Comment:
Please fix `server` and `provider-server` naming. This image may not be necessary `provider-server`. Shall we call it `cardano-services`?

Reply:
If there's just a single image for all services then I agree

Reply:
Ok, this is due to the re-architecture. What's the name, then? `cardano-sdk-server`?

Reply:
If this image is being used to run instances of `provider-server`, `projector`, `pgboss-worker` etc, it's the appropriate to call it by the package name: `cardano-services`

Reply:
Implemented in 0b0b91f153c7d232ac10a797ad646f2f26950e23